.gitignore

```diff
@@ -7,7 +7,7 @@ docs/index.md
 lerna-debug.log
 node_modules
 package-lock.json
-packages/**/build
+packages/@snowpack/**/build
```

Comment:
So funny enough we were ignoring `packages/snowpack/src/build` 😅. Oops! (no wonder I could never jump straight to `build-pipeline.ts`

Reply:
i was wondering that too!